src/java/org/apache/cassandra/schema/ColumnMetadata.java

```diff
@@ -275,6 +319,11 @@ public boolean isMasked()
         return mask != null;
     }
 
+    public boolean hasConstraint()
+    {
+        return columnConstraints != null && !columnConstraints.isEmpty();
```

Comment:
@bbotella why do you check for nullity if you put Nonnull annotations on various places? How can it happen that columnConstraints are null?

Reply:
You are right. We can get rid of those by touching a couple lines on TableMetadata.